tools/resourcedocsgen/cmd/metadata.go

```diff
@@ -73,6 +73,73 @@ var categoryLookup = map[string]pkg.PackageCategory{
 	"sumologic":     pkg.PackageCategoryMonitoring,
 }
 
+// TODO[pulumi/pulumi#7813]: Remove this lookup once display name is available in
+// the Pulumi schema.
+// titleLookup is a map to map module package name to the desired display name
+// for display in the TOC menu under API Reference.
+// NOTE: For the time being this lookup map and the one used by the docs
+// generator in `pulumi/pulumi` must be kept up-to-date.
+var titleLookup = map[string]string{
```

Comment:
I am sorry for this but it's needed without need to make more changes in the docs generator in `pulumi/pulumi` to use the existing title lookup here. In any case, both will be removed when pulumi/pulumi#7813 is implemented.

Reply:
@infin8x fyi.

Reply:
Ah nice, thanks! Once this PR is in, can I propose some changes to these titles? (Or can I just do it here? Don't want to get in the way at all)

Reply:
Absolutely, please feel free to do it here, either as a comment or as a GH suggestion against the appropriate lines and I'll accept them.

Reply:
Perfect, done.